interview-questions.md

```diff
@@ -6,40 +6,60 @@ First, without external resources. Challenge yourself to answer from memory.
 
 Then, research the question to expand on your answer. Even if you feel you have answered the question completely on your own, there is always something more to learn. Write your researched answer in your OWN WORDS.
 
-Type your initials here to confirm you read the instructions:
+Type your initials here to confirm you read the instructions: DLD 
 
 1. As a Rails developer, I am creating an application with a model called Cohort and a model called Student. The relationship between these models is that a cohort has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-Your answer:
+Your answer: i honestly dont know without looking up
 
-Researched answer:
+Researched answer: generate a migration to add the key to Student research says I should call it AddCohortReferenceToStudents and then pass it cohort:references. then db:migrate to save the changes. And now i can add the has_many to cohort and belongs_to to student. the foreign key name is cohort:id and the foreign key will be in the Student model.
```

Comment:
Saw your note asking for an explanation. Your research is correct. In order to add a column to an existing model, we need to generate a migration, which will create a new migration file. In that file, we can then specify the changes we want using a method within the "AddCohortReferenceToStudents" class, to go with your example. To apply this change to the existing model, we then need to run db:migrate. The point of this question is to see if you understand that a model can't be changed by simply going into the schema and adding the foreign key manually in there, as schema's are read-only.